packages/matter.js/src/behavior/definitions/device-energy-management/DeviceEnergyManagementServer.ts

```diff
@@ -15,4 +14,10 @@ import { DeviceEnergyManagementBehavior } from "./DeviceEnergyManagementBehavior
  * default. You should use {@link DeviceEnergyManagementServer.with} to specialize the class for the features your
  * implementation supports.
  */
-export class DeviceEnergyManagementServer extends DeviceEnergyManagementBehavior {}
+export class DeviceEnergyManagementServer extends DeviceEnergyManagementBehavior {
+    override initialize() {
+        // Technically ElectricalSensor also requires Device EnergyManagementMode (as of Matter 1.3) but no other device
+        // type requires DeviceEnergyManagement
+        this.agent.get(DescriptorServer).addDeviceTypes("ElectricalSensor");
```

Comment:
Formally check that a PowerTopology cluster also exists on the endpoint because that one is the mandatory one?

Reply:
Meant to revert this file